README.md

```diff
@@ -293,7 +293,7 @@ The default is `false`.
 
 ### `shell` (optional, run only)
 
-The shell that is used to invoke commands. This is so that multiple commands can be interpretted.
+The shell that is used to invoke commands. This is so that multiple commands can be interpretted. if `false` is specified, the command is invoked directly.
```

Comment:
The shell that is used to invoke commands. Set to `false` to disable running the commands via a shell (useful for images that expect command arguments to passed directly).